app/models/address.rb

```diff
@@ -0,0 +1,5 @@
+class Address < ApplicationRecord
+  belongs_to :addressable, polymorphic: true, optional: true
```

Comment:
Why optional?

Reply:
You're right, it was a temp bypass to allow the creation of address without the need to have an associated `addressable` in dev, will remove it